src/components/modal/modal.tsx

```diff
@@ -106,21 +114,32 @@ export const Modal: FC<ModalProps> = ({
             exit={{ opacity: 0, marginTop: -modalMargin }}
             transition={{ duration: 0.3 }}
           >
-            <ModalHeader title={title} headerNode={headerNode} onClose={closeModal} />
+            <ModalHeader
+              title={title}
+              headerNode={headerNode}
+              onClose={closeModal}
+              headerContentClassName={headerContentClassName}
+              headerDescription={headerDescription}
+            />
             {scrollable ? (
               <Scrollbars autoHeight autoHeightMax={contentMaxHeight} hideTracksWhenNotNeeded>
                 <ModalContent>{children}</ModalContent>
               </Scrollbars>
             ) : (
               <ModalContent>{children}</ModalContent>
             )}
-            <ModalFooter
-              size={size}
-              footerNode={footerNode}
-              okButton={okButton}
-              cancelButton={cancelButton}
-              closeHandler={closeModal}
-            />
+            {!withoutFooter &&
```

Comment:
Do we need to adjust scroll area height calculation in case of no footer?
I suppose we also need to specify some restriction for custom footer size to not break the scroll area calculation logic. What do you think?